### ISSUE TITLE
Questions about ResBlock implementation

### ISSUE THREAD

**Issue by doutzi** [genforce/idinvert_pytorch#14]
Hi, first of all thanks for the great work!

I just compared the ResBlock implementation with the original tensorflow implementation and I'm a little bit confused. Shouldn't there be an activation layer (e.g. LeakyReLU) after the first convolutional block? And the WScaleLayer used within the ResBlock, does this have the same impact as in the StyleGAN implementation, i.e. enables equalized learning rate if use_wscale=True?

Thank you for the clarification!

**Comment by zhujiapeng** [comment]
This is the first version of our encoder structure, the weights give here are exactly match the encoder's structure here.  We updated the TensorFlow version a little from the first version.
